Library/Formula/mandoc.rb

```diff
@@ -0,0 +1,74 @@
+class Mandoc < Formula
+  homepage "http://mdocml.bsd.lv"
+  url "http://mdocml.bsd.lv/snapshots/mdocml-1.13.3.tar.gz"
+  sha256 "23ccab4800d50bf4c327979af5d4aa1a6a2dc490789cb67c4c3ac1bd40b8cad8"
+
+  head "anoncvs@mdocml.bsd.lv:/cvs", :module => "mdocml", :using => :cvs
+
+  depends_on "sqlite"
```

Comment:
It wants SQLite 3.8.4.3 or newer. I wasn't sure what version 10.6/10.7 ships with, since (bizarrely) all the links at http://opensource.apple.com/source/SQLite are broken. Do you happen to know offhand?